app/Sidebar.tsx

```diff
@@ -171,6 +171,10 @@ const MenuLink = React.forwardRef<
       <NavigationMenu.Link active={isActive} asChild>
         <Link
           href={href}
+          onClick={() => {
+            // @see https://github.com/framer/motion/issues/2006#issuecomment-1477824846
+            window.scroll(0, 0);
```

Comment:
Can you remove the semicolon at the end? Thanks!

Reply:
It looks like prettier is not working for me